### PR TITLE
Likert scale

### PR DESCRIPTION
**Pull request by tomprats**

### CODE REVIEW

File: src/components/slide-deck/index.js [hunk 1]
```diff
@@ -209,12 +232,14 @@ class SlideDeck extends Component {
             instructions={this.state.instructions}
             isComplete={finished}
             isFullscreen={this.state.isFullscreen}
+            isLikertScale={this.props.assessment && this.props.assessment.scoring_scale === "LIKERT_CUMULATIVE_POMP"}
```
Comment (Jbur43) [inline src/components/slide-deck/index.js]: Could you only check for the scoring scale here if the first condition is handling loading?

File: src/components/slide-deck/helpers.js [hunk 2]
```diff
@@ -40,13 +41,15 @@ export function isFullscreen() {
...
+  return slides.findIndex((slide) => (
+    !slide.loaded && slide.likert_response == null && slide.response == null
+  ));
 }
 
 export function mutable(data) { return data.map((item) => ({...item})); }
 
 export function slideIndex(slides) {
-  return slides.findIndex((slide) => (slide.response == null));
+  return slides.findIndex((slide) => (slide.likert_response == null && slide.response == null));
```
Comment (Jbur43) [inline src/components/slide-deck/helpers.js]: For the responses could this filter be applied on all falsy values?

File: src/components/slide-deck/helpers.js [hunk 3]
```diff
@@ -40,13 +41,15 @@ export function isFullscreen() {
 }
 
 export function loadingIndex(slides) {
-  return slides.findIndex((slide) => (!slide.loaded && slide.response == null));
+  return slides.findIndex((slide) => (
+    !slide.loaded && slide.likert_response == null && slide.response == null
```
Comment (Jbur43) [inline src/components/slide-deck/helpers.js]: For the responses could this filter be applied on all falsy values?

File: src/components/slide-deck/helpers.js [hunk 4]
```diff
@@ -1,8 +1,9 @@
 export function completedSlides(slides) {
-  return slides.filter(({response}) => (
-    response != null
-  )).map(({id, response, time_taken: timeTaken}) => ({
+  return slides.filter(({likert_response: likertResponse, response}) => (
+    likertResponse != null || response != null
```
Comment (Jbur43) [inline src/components/slide-deck/helpers.js]: For the responses could this filter be applied on all falsy values?

File: src/components/slide-deck/index.js (hunk 1 above)
Comment (tomprats) [reply src/components/slide-deck/index.js]: Good point +1

**Pull request merged**